src/announce.transactions.command.ts

```diff
@@ -27,7 +29,14 @@ import {PasswordValidator} from './validators/password.validator';
  * Base command class to announce transactions.
  */
 export abstract class AnnounceTransactionsCommand extends ProfileCommand {
-
+    private table: HorizontalTable;
```

Comment:
Could we create the table in a separate class? Like we are doing in every other command, see https://github.com/nemtech/nem2-cli/blob/master/src/commands/account/info.ts#L48